aggregator_core/src/datastore/models.rs

```diff
@@ -362,7 +362,6 @@ pub struct AggregationJob<const SEED_SIZE: usize, Q: QueryType, A: vdaf::Aggrega
     /// The ID of this aggregation job.
     aggregation_job_id: AggregationJobId,
     /// The aggregation parameter this job is run with.
-    #[derivative(Debug = "ignore")]
```

Comment:
I think we made this change because the aggregation parameter was potentially sensitive (e.g., IIUC, for Poplar1/Mastic it would store prefixes of measurement values); I think we want to avoid logging aggregation parameter values.

If we really need to log these, maybe we could:
a) log some safe value derived from the aggregation parameter, e.g. a hash of the value
b) require that the `Debug` implementations of aggregation parameters take responsibility for producing non-sensitive debug values in a VDAF-specific way

(b) is the best IMO but also the hardest to implement, and we'd need to be careful not to backslide as new VDAF implementations come around.

Reply:
I did this because I wanted to see what the aggregation param value being used with the dummy VDAF is, and that type is simple enough that including it in Debug output wasn't too noisy. I agree that (b) is desirable, but currently `Poplar1AggregationParam`'s debug output contains too much noise. So I'm going to revert this.